src/platform_impl/linux/wayland/window.rs

```diff
@@ -334,8 +334,12 @@ impl Window {
 
     pub fn fullscreen(&self) -> Option<Fullscreen> {
         if *(self.fullscreen.lock().unwrap()) {
+            // We're reporting incorrect value to a user to prevent a crash. In general
+            // we can't do much about it unless we'd be able to indicate that we're in a
+            // fullscreen state, but we don't know on which output.
```

Comment:
This seems very verbose.

```suggestion
            // If the monitor cannot be determined, we cannot report any fullscreen mode.
```

Seems like this has the same amount of information, in significantly less words? I don't see much point in the second sentence of your documentation.

Reply:
That's some sort of meta `TODO`, I'm fine with shortening it, since I won't forget to properly rewrite it once #1670 will be ready.